core/sessions/localauth/reaper_test.go

```diff
@@ -53,32 +55,34 @@ func TestSessionReaper_ReapSessions(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			user := cltest.MustRandomUser(t)
-			require.NoError(t, orm.CreateUser(&user))
-
-			session := sessions.NewSession()
-			session.Email = user.Email
-
-			_, err := db.Exec("INSERT INTO sessions (last_used, email, id, created_at) VALUES ($1, $2, $3, now())", test.lastUsed, user.Email, test.name)
-			require.NoError(t, err)
-
 			t.Cleanup(func() {
-				_, err2 := db.Exec("DELETE FROM sessions where email = $1", user.Email)
-				require.NoError(t, err2)
+				clearSessions(t, db.DB)
```

Comment:
I think this edit is wrong: this should just delete the user that was created. Doing a blanket "DELETE FROM" will cause lock contention on the sessions table and may make this test flakey.

Reply:
Good catch, updated 🙂